docs/examples/fd_fitting_basic/index.rst

```diff
@@ -0,0 +1,113 @@
+Twistable Worm-Like-Chain Fitting
+=================================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+First we import the necessary libraries::
+
+    import matplotlib.pyplot as plt
+    from lumicks import pylake
+    import numpy as np
+    import h5py
+
+Load the data from disk
+-----------------------
+
+This is pre Bluelake data, therefore we have to extract the data from the h5
```

Comment:
I'm not entirely sure to be honest. This was data from Onno's F, d package, so I wouldn't be surprised if it was from pre-lumicks. I used it as test for the tWLC. I mean, we could do two things. Omit the example. Repack the data in a faux bluelake format. Then again, we don't ship the data with the examples, so I could just adjust the example as though it was in a bluelake format. Maybe that's the best option.

Reply:
It would be ideal to move non-Bluelake data to a separate section and include pure `pylake` examples here.

Reply:
Yes. I mean, I didn't really intend for these to be docs in the first place. Originally they were Jupyter notebooks on the PR. As an alternative, I suggest just leaving the examples out completely and just have the tutorial, since we have no actual Bluelake data for them. After May fourth, maybe we can add the RecA data.

The tWLC is old Onno data, the RecA is old Jordi data and the AdK relies on preprocessing with a piezo script that's not part of pylake and not ready for release.